src/Survey/index.jsx

```diff
@@ -0,0 +1,228 @@
+/** @format */
+
+import "./survey.css";
+import titleCard from "../Assets/redBanner.png";
+import { Button } from "../Components/Buttons/Button";
+import { Radio } from "../Components/RadioButton/Radio";
+import { useEffect, useState } from "react";
+import { useNavigate } from "react-router-dom";
+import validator from "validator";
+import UseLocalStorage from "../Components/UseLocalStorage";
+
+const Survey = () => {
+	const [isDisabled, setIsDisabled] = useState(false);
+	const [names, setNames] = useState("");
+	const [age, setAge] = useState(0);
+	const [email, setEmail] = useState("");
+	const [gender, setGender] = useState("Male");
+	const [submitSuccess, setSubmitSuccess] = useState(false);
+
+	const navigate = useNavigate();
+
+	const [surveyData, setSurveyData] = UseLocalStorage("Survey_Taken_By", "");
+
+	const [error, setError] = useState({
+		names: "",
+		email: "",
+		age: "",
+	});
+
+	const showData = () => {
+		console.log("Name: ", names);
+		console.log("Age: ", age);
+		console.log("Gender: ", gender);
+		console.log("Email: ", email);
+	};
+	const regex =
+		/^(([^<>()[\]\.,;:\s@\"]+(\.[^<>()[\]\.,;:\s@\"]+)*)|(\".+\"))@(([^<>()[\]\.,;:\s@\"]+\.)+[^<>()[\]\.,;:\s@\"]{2,})$/i;
+
+	const onSubmit = (e) => {
+		e.preventDefault();
```

Comment:
Need better and clean logic for form validation.

Reply:
Resolved